app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTrayInteractor.kt

```diff
@@ -30,6 +30,11 @@ interface BrowserTrayInteractor : SelectionInteractor<Tab>, UserInteractionHandl
      * TabTray's Floating Action Button clicked.
      */
     fun onFabClicked(isPrivate: Boolean)
+
+    /**
+     * TabTray's Inactive tabs -> Recently Closed clicked.
```

Comment:
Maybe we move this to a different place in the tabs tray tomorrow. 🤷 
```suggestion
     * Recently Closed item is clicked. 
```

Reply:
Good point.